src/build/index.js

```diff
@@ -1,16 +1,31 @@
 'use strict'
 
-const build = require('./browser')
-const experimentalBuild = require('./experimental-browser')
-const debug = require('debug')('aegir:build')
+const path = require('path')
+const execa = require('execa')
+const rimraf = require('rimraf')
+const { fromAegir } = require('./../utils')
 
-module.exports = (argv) => {
-  debug('argv', argv)
-  if (argv['enable-experimental-browser-builds']) {
-    debug('running experimental build')
-    return experimentalBuild(argv)
-  }
+// Clean dist
+rimraf.sync(path.join(process.cwd(), 'dist'))
```

Comment:
yes

Reply:
Perhaps I don't know enough about the code base yet, could you explain why this is done when this file is first required and not when the export is executed? In general I'd not expect side effects from requiring a file unless it's a script that I'd execute from node directly.

Reply:
you'r right this should be inside the export, in this cause it wouldn't do bad things because we require this file inside the cmd handler but its definitely more idiomatic inside.
i have push a commit fixing it. Sorry should have paid more attention, too many issues and comments :)